osquery/tests/test_util.cpp

```diff
@@ -37,14 +37,17 @@ std::string kFakeDirectory = "";
 #ifdef DARWIN
 std::string kTestWorkingDirectory = "/private/tmp/osquery-tests";
 #else
-std::string kTestWorkingDirectory = "/tmp/osquery-tests";
+std::string kTestWorkingDirectory =
```

Comment:
We're being inconsistent with strings and path objects within tests. Shouldn't we avoid complex initialization in global scope? Or at least try to be consistent.